ecl/hql/hqlutil.hpp

```diff
@@ -197,6 +197,8 @@ extern HQL_API IHqlExpression * createTransformForField(IHqlExpression * field,
 extern HQL_API IHqlExpression * convertScalarToRow(IHqlExpression * value, ITypeInfo * fieldType);
 extern HQL_API bool splitResultValue(SharedHqlExpr & dataset, SharedHqlExpr & attribute, IHqlExpression * value);
 
+//Is this really dependent on a parameter - expr->isFullyBound() can give false positives.
```

Comment:
This comment is not as clear, perhaps it should belong with (or copied to) the func def?